src/Features/CSharp/Portable/Completion/CompletionProviders/CrefCompletionProvider.cs

```diff
@@ -291,6 +291,13 @@ private static TextSpan GetTextChangeSpan(SourceText text, int position)
                 .Replace('>', '}')
                 .ToString();
 
+            // Sorting on symbolText or insertionText would have an annoying result for types like Action or Tuple.
+            // See issue: https://github.com/dotnet/roslyn/issues/8623
+            var sortText = builder
+                .Replace('{', ' ')
+                .Replace('}', ' ')
+                .ToString();
```

Comment:
I wonder what the rest of @dotnet/roslyn-ide thinks of this issue I mentioned on the bug:

> We also can't use "whitespace and identifier character strings with type parameter names" either:
> 
> genericType ztypeparameter would be sorted after generictype atypeparameter btypeparameter.

If we can't use the metadata name, can we take the type name and append the number of type parameters with some leading 0 padding?

Reply:
💭 Seems like the sort text should use the metadata name (with arity, e.g. ``Dictionary`2``) and just ignore all the type parameters. Not sure if that would break other scenarios though.